src/orders/components/OrderChangeWarehouseDialog/OrderChangeWarehouseDialog.tsx

```diff
@@ -184,12 +194,19 @@ export const OrderChangeWarehouseDialog: React.FC<OrderChangeWarehouseDialogProp
             })}
           </RadioGroup>
         ) : (
-          <Skeleton />
+          <div className={classes.loader}>
+            <CircularProgress />
+          </div>
```

Comment:
I need to check it with @pwgryglak but almost all components use skeletons and it makes this component inconsistent with the rest